docs-ref-conceptual/dotnet-howto-migrate-app-service.md

```diff
@@ -109,7 +109,7 @@ If your app cannot be migrated directly to App Service, consider App Service usi
 * [Moving your database to the cloud](https://go.microsoft.com/fwlink/?linkid=863217)
 * [Azure Web App sandbox details and restrictions](https://github.com/projectkudu/kudu/wiki/Azure-Web-App-sandbox)
 
-## Next steps
+## Next Steps
```

Comment:
I know it seems wrong, but titles only capitalize the first word according to the [MSFT style guide](https://worldready.cloudapp.net/Styleguide/Read?id=2696&topicid=43940).  It looks to me like you changed this to be consistent with the other H2s, but it looks like "Next steps" was correct and the other H2s in the article are wrong.

I'm glad you noticed, though - I just updated this article for something else and I hadn't noticed!

Reply:
Oh no, I totally get that point, but to clarify, this tab isn't for tools *targeting* Linux, it's for tools that *run* on Linux.

I think it certainly speaks to a need for a clarification on that point, though.

Reply:
And actually, if you have any insights about the target audience or relevant scenarios, that would be helpful since I have more doc updates to look at elsewhere. What I know right now is that Python and Node.js devs will often want Linux deployment if they're using VS. For .NET, it's a benefit for folks that already have Linux experience and want to use it. Other scenarios--not as clear.

Reply:
Ah, got it.